attic/platform.py

```diff
@@ -8,6 +10,33 @@
     from attic.platform_freebsd import acl_get, acl_set, API_VERSION
 elif platform == 'Darwin':
     from attic.platform_darwin import acl_get, acl_set, API_VERSION
+elif platform.startswith('CYGWIN'):
+    API_VERSION = 2
+
+    def acl_get(path, item, st, numeric_owner=False):
+        try:
+            # Using non-numeric names, i.e., group names, has caused problems.
+            # Hence the -n option
+            ACL_text = subprocess.check_output(['getfacl.exe', '-n', path])
```

Comment:
how's performance with executing getfacl once per file you backup? is there a windows api one could call from python?